### ISSUE TITLE
Deploy duty

### ISSUE THREAD

**Issue by cdanfon** [MozillaFoundation/foundation.mozilla.org#9744]
Ongoing ticket to deploy work to production.

**When?**

**Tuesday** morning (and potentially other times). 

Be on the look out for other times when there is work in the "Ready for production" column and a deployment is not blocked by untested work in the "QA" column. That is a good time to deploy to production.

**Steps** 

- Before deploying, check that all work on main is on staging, has been QA'd and does not contain any release blocking issues. 
- You should be able to get this information from the Zenhub board. The "QA" column should be empty. 
- If not, try to get those tickets QA'd and moved to "Ready for production" before deploying.
- If all is well login to Heroku and promote the staging app to production.
- Move all tickets on the Zenhub board from "Ready for production" to "Closed"

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/TP1-311)

**Comment by mmmavis** [comment]
Not sure where to best flag this. There are a few [PNI regressions](https://github.com/mozilla/foundation.mozilla.org/issues/10059) on staging that need to be fixed before the next prod push.

**Comment by tbrlpld** [comment]
@mmmavis Would be best to add the "deployment blocker" label to all these tickets.

**Comment by mmmavis** [comment]
@tbrlpld Got it! And thanks for mentioning this on Slack so everyone knows.

In this case I don't know which code/PR was causing the regression. It will be for the PR authors to label their corresponding tickets.

**Comment by tbrlpld** [comment]
In that case maybe we need a bug ticket for the regression and that should have the label then.

**Comment by mmmavis** [comment]
@tbrlpld yup yup it's this one https://github.com/mozilla/foundation.mozilla.org/issues/9744#issuecomment-1405784945 :)

**Comment by tbrlpld** [comment]
@mmmavis That is the deploy duty ticket

**Comment by mmmavis** [comment]
> @mmmavis That is the deploy duty ticket 

@tbrlpld ah the link might not be obvious - it brings you to my comment above that includes a "PNI regressions" link which then brings you to this regression ticket https://github.com/mozilla/foundation.mozilla.org/issues/10059 (already labelled with "deployment blocker" )
